README.md

```diff
@@ -1,5 +1,7 @@
 # Cass Operator
 [![License: Apache License 2.0](https://img.shields.io/github/license/k8ssandra/cass-operator)](https://github.com/k8ssandra/cass-operator/blob/master/LICENSE.txt)
+[![Gitter](https://badges.gitter.im/cass-operator/community.svg)](https://gitter.im/cass-operator/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
```

Comment:
@bradfordcp this is yet another communication channel to add to the list.  Looks like there are discussions on here as recently as late April, so seems active.  What do we want to do here in our consolidation efforts?